mirrord-operator/values.yaml

```diff
@@ -15,7 +15,8 @@ roleNamespaces: []
 
 operator:
   image: ghcr.io/metalbear-co/operator
-
+  podAnnotations: {}
+  podLabels: {}
```

Comment:
maybe podExtraLabels because `podLabels` feels it should change the selector as well (in service too)

Reply:
Most helm charts adhere to this name so I don't think it's good enough reason to break the convention
https://github.com/goharbor/harbor-helm/blob/main/templates/nginx/deployment.yaml